### PR TITLE
Added some hotstrings

### PR DESCRIPTION
**Pull request by akshat-2110**
After merging change last hosting which include my details

### CODE REVIEW

File: auto_correct.ahk [hunk 1]
```diff
@@ -5693,6 +5693,6 @@ return  ; This makes the above hotstrings do nothing so that they override the i
 ::chacne::chance
 
 ::systemc::SystemC
-::vishal::Vishal
-::chovatiya::Chovatiya
+::akshat::Akshat
```
Comment (vishalchovatiya) [inline auto_correct.ahk:5698]: Overwriting changes relevant to me.

File: hotstrings.ahk [hunk 2]
```diff
@@ -1,35 +1,67 @@
...
-::bcoz::because,
 
-::aka::also known as
+::bcoz::because,
 
-::br::BR,{Enter}Vishal Chovatiya
-::heym::Hi,{Enter}{Enter}{Enter}BR,{Enter}Vishal Chovatiya{Enter}{left 25}
-::rslot::Hi,{Enter}{Enter}Reserving the slot for the same.{Enter}{Enter}BR,{Enter}Vishal Chovatiya{Enter}{left 58}
-::pat::Hi,{Enter}{Enter}{Enter}{Enter}Purpose:{Enter}-{Space}{Space}{Enter}{Enter}{Enter}Action:{Enter}-{Space}{Space}{Enter}{Enter}{Enter}Timeline:{Enter}-{Space}{Space}{Enter}{Enter}{Enter}BR,{Enter}Vishal Chovatiya{Enter}{Enter}{left 66}
+::br::BR,{Enter}Akshat Singhal
```
Comment (vishalchovatiya) [inline hotstrings.ahk]: Same here.

File: hotstrings.ahk [hunk 3]
```diff
@@ -1,35 +1,67 @@
...
+::heym::Hi,{Enter}{Enter}{Enter}{Enter}BR,{Enter}Akshat Singhal{Enter}{left 21}
+::rslot::Hi,{Enter}{Enter}Reserving the slot for the same.{Enter}{Enter}BR,{Enter}Akshat Singhal{Enter}{left 58}
+::pat::Hi,{Enter}{Enter}{Enter}{Enter}Purpose:{Enter}-{Space}{Space}{Enter}{Enter}{Enter}Action:{Enter}-{Space}{Space}{Enter}{Enter}{Enter}Timeline:{Enter}-{Space}{Space}{Enter}{Enter}{Enter}BR,{Enter}Akshat Singhal{Enter}{Enter}{left 66}
 
 ::ttl::In middle of something, will talk to you soon
 ::mttl::In meeting, will come back to you soon
+::ttyl::talk to you later
+
+::mygmail::akshatsinghal2110@gmail.com
+::myoutlook::akshat.singhal@infineon.com
```
Comment (vishalchovatiya) [inline hotstrings.ahk]: This also.

**Review by vishalchovatiya — CHANGES_REQUESTED** [review]
There are some changes that are personal to the user.

File: hotstrings.ahk (hunk 3 above)
Comment (akshat-2110) [reply hotstrings.ahk]: Changes done

**Pull request closed without merging**